readme.md

```diff
@@ -1,152 +1,160 @@
-[benchmark] server-console-block   | Iters 1000000 | Elapsed 15.784 secs     63354/sec
-
-[benchmark] base-file-overrun      | Iters 1000000 | Elapsed  0.439 secs   2273952/sec
-[benchmark] daily-file-overrun     | Iters 1000000 | Elapsed  0.451 secs   2212609/sec
-[benchmark] rotating-file-overrun  | Iters 1000000 | Elapsed  0.453 secs   2204658/sec
-[benchmark] server-console-overrun | Iters 1000000 | Elapsed  0.372 secs   2684282/sec
-
-
-# Queue size: 131072    Thread count: 4
-[benchmark] base-file-block        | Iters 1000000 | Elapsed   0.998 secs   1001216/sec
-[benchmark] daily-file-block       | Iters 1000000 | Elapsed   0.973 secs   1027070/sec
-[benchmark] rotating-file-block    | Iters 1000000 | Elapsed   0.956 secs   1045255/sec
-[benchmark] server-console-block   | Iters 1000000 | Elapsed  13.952 secs     71671/sec
-
-[benchmark] base-file-overrun      | Iters 1000000 | Elapsed   0.472 secs   2116635/sec
-[benchmark] daily-file-overrun     | Iters 1000000 | Elapsed   0.441 secs   2264892/sec
-[benchmark] rotating-file-overrun  | Iters 1000000 | Elapsed   0.478 secs   2091503/sec
-[benchmark] server-console-overrun | Iters 1000000 | Elapsed   0.385 secs   2592245/sec
-
-
-# Queue size: 8192      Thread count: 8
-[benchmark] base-file-block        | Iters 1000000 | Elapsed  1.135 secs    881010/sec
-[benchmark] daily-file-block       | Iters 1000000 | Elapsed  1.183 secs    845069/sec
-[benchmark] rotating-file-block    | Iters 1000000 | Elapsed  1.193 secs    838199/sec
-[benchmark] server-console-block   | Iters 1000000 | Elapsed 14.925 secs     67000/sec
-
-[benchmark] base-file-overrun      | Iters 1000000 | Elapsed  0.533 secs   1875363/sec
-[benchmark] daily-file-overrun     | Iters 1000000 | Elapsed  0.569 secs   1754767/sec
-[benchmark] rotating-file-overrun  | Iters 1000000 | Elapsed  0.508 secs   1967969/sec
-[benchmark] server-console-overrun | Iters 1000000 | Elapsed  0.394 secs   2532556/sec
-```
-
-#### Sourcemod logging
-
-As a reference, [sourcemod logging API](https://sm.alliedmods.net/new-api/logging) was also tested
-
-
-```
-[benchmark] LogMessage    | Iters 1000000 | Elapsed 10.740 secs     93108/sec
-[benchmark] LogToFile     | Iters 1000000 | Elapsed  9.091 secs    109989/sec
-[benchmark] LogToFileEx   | Iters 1000000 | Elapsed  8.823 secs    113336/sec
-[benchmark] PrintToServer | Iters 1000000 | Elapsed  5.779 secs    173024/sec
-```
-
-## Credits
-
-- **[gabime](https://github.com/gabime) [spdlog](https://github.com/gabime/spdlog)** library implements most of the functionality, log4sp.ext wraps the spdlog API for SourcePawn to use
-
-- Fyren, nosoop, Deathreus provides solution for managing the Sink Handle
-
-If I missed anyone, please contact me.
-
+**[English](./readme.md) | [中文](./readme-chi.md)**
+
+# Logging for SourcePawn
+
+This is a Sourcemod extension that wraps the [spdlog](https://github.com/gabime/spdlog) library to enhance SourcePawn logging and debugging.

```

Comment:
The documentation should be updated. log4sp is more of an adapter than a wrapper.